app/frontend/app/controllers/anime/quotes.js

```diff
@@ -37,6 +30,7 @@ export default Ember.ArrayController.extend(HasCurrentUser, {
           });
 
       quote.save();
+      this.get('content').addObject(quote);
```

Comment:
I just tested it without that line, it doesn't get automatically pushed to the model. I guess an alternative way would be using `model.refresh();` but that's triggering all the model hooks again so I think this is the best solution for handling a simple object-push.